# Create this message for CANFD
Header header
uint32 id
bool is_rtr
bool is_extended
bool is_error
uint32 dlc
uint8[] data